be/src/olap/compaction.cpp

```diff
@@ -642,6 +642,9 @@ Status Compaction::do_inverted_index_compaction() {
     // format: rowsetId_segmentId
     std::vector<std::unique_ptr<InvertedIndexFileWriter>> inverted_index_file_writers(
             dest_segment_num);
+
+    // Some columns have already been indexed
+    std::vector<int64_t> compacted_idx_file_size(dest_segment_num, 0);
```

Comment:
It is prone to errors to using index of vector to record size. I suggest you use a map with index file path as key.

Reply:
index file name ?

Reply:
`compacted_idx_file_size[i]` corresponds to `inverted_index_file_writers[i]`

Reply:
done